circuit/src/integer/rns.rs

```diff
@@ -313,6 +309,25 @@ impl RnsParams<Fq, Fr, 4, 68> for Bn256_4_68 {
 		(q, result)
 	}
 
+	fn construct_mul_qr(a_bn: BigUint, b_bn: BigUint) -> ([Fr; 4], [Fr; 4]) {
+		let wrong_mod_bn = Self::wrong_modulus();
+		let (quotient, result_bn) = (a_bn * b_bn).div_rem(&wrong_mod_bn);
+		let q = decompose_big::<Fr, 4, 68>(quotient);
+		let result = decompose_big::<Fr, 4, 68>(result_bn);
+		(q, result)
+	}
+
+	fn construct_div_qr(a_bn: BigUint, b: Integer<Fq, Fr, 4, 68, Self>) -> ([Fr; 4], [Fr; 4]) {
```

Comment:
Can we have a consistent interface here?
We should pass `a_bn: BigUint, b: BigUint, b_invert: BigUint`.
And also add a check `assert!(b * b_invert == 1)`